docs/core-concepts/networking.md

```diff
@@ -125,6 +125,7 @@ Each host Node runs a number of infrastructure services as Docker containers, us
 | Weave Net | TCP      | 6783 | `*`                       | Weave Net Control
 | Weave Net | UDP      | 6783 | `*`                       | Weave Net Data (`sleeve`)
 | Weave Net | UDP      | 6784 | `*`                       | Weave Net Data (`fastdp`)
+| Weave Net | ESP      | 6784 | `*`                       | Weave Net Data (`IPSec fastdp`)
```

Comment:
I'd mark the Protocol as `ESP + UDP` or `ESP (UDP)` or somesuch... ESP + a port number doesn't make sense :)